src/goto-programs/goto_program.h

```diff
@@ -814,6 +814,13 @@ class goto_programt
     return instructiont(code_returnt(), irep_idt(), l, RETURN, nil_exprt(), {});
   }
 
+  static instructiont make_return(
+    code_returnt c,
+    const source_locationt &l = source_locationt::nil())
+  {
+    return instructiont(std::move(c), irep_idt(), l, RETURN, nil_exprt(), {});
```

Comment:
We need to make progress on #3126, which removes all these `irep_idt()` in the `make_*` functions.

Reply:
Indeed!